code/sorting/Sleep_Sort/Sleep_sort.cpp

```diff
@@ -0,0 +1,17 @@
+#include <unistd.h>
```

Comment:
Kindly consult this [C code](https://github.com/OpenGenus/cosmos/blob/master/code/sorting/Sleep_Sort/Sleep_Sort.c) and make necessary changes 👍

Reply:
what's wrong with using unistd?

Reply:
Nevermind; I updated to use standard C++11 functions 😄